src/parts/_misc.css

```diff
@@ -38,7 +38,7 @@ tbody tr:nth-child(even) {
   border-radius: 6px;
 }
 
-::-webkit-scrollbar-thumb {
+::-webkit-scrollbar-thumb {
```

Comment:
There's a weird character between the `b` and `a` in `-scrollbar`:
![image](https://user-images.githubusercontent.com/6742479/58606670-4e5a1880-8251-11e9-8dad-f76d3e94ec58.png)